packages/edit-site/src/components/dataviews/view-list.js

```diff
@@ -132,18 +132,12 @@ function ViewList( {
 	paginationInfo,
 } ) {
 	const columns = useMemo( () => {
-		const _columns = [
-			...fields.map( ( field ) => {
-				const column = { ...field };
-				delete column.render;
-				column.cell = ( props ) => {
-					return field.render
-						? field.render( { item: props.row.original, view } )
-						: field.accessorFn( props.row.original );
-				};
-				return column;
-			} ),
-		];
+		const _columns = fields.map( ( field ) => {
+			const column = { ...field };
+			delete column.render;
+			column.cell = ( props ) => field.render( props.row.original, view );
```

Comment:
While we're copying, why not

```suggestion
			const { render, ...column } = field;
			column.cell = ( props ) => render( props.row.original, view );
```